### DISCUSSION TITLE
Asset Report Vendor Incompatibility - Thoughts?

### DISCUSSION THREAD

**Discussion by deefour** [TomorrowIdeas/plaid-sdk-php#54] [Q&A]
We're having issues with [a vendor](https://ocrolus.com) accepting asset report JSON from this library. 

```
require __DIR__ . '/vendor/autoload.php';

$client = new \TomorrowIdeas\Plaid\Plaid('key', 'secret', "production");

$payloadForVendor = json_encode(
    $client->reports->getAssetReport('asset-report-token', true)
);
```

I'm looking for advice; not creating an issue yet because I'm not convinced this is a problem with this library yet. [The vendor docs suggest](https://docs.ocrolus.com/reference/upload-plaid-json-to-book) they've had issues with some official libraries too. I've separately reached out to them.

If I fetch the asset report JSON directly from plaid via Postman/cURL, the vendor accepts the data without issue.

I've tried diffing the JSON from this SDK and the raw data from Plaid, but there is too much reorganizing going on to immediately pinpoint a cause.

Thoughts?

**Comment by brentscheffler** [comment]
This library simply passes through the raw response from Plaid as an object without modifying or decorating it. If possible, can you paste the response from the `getAssetReport` call (obfuscating any sensitive data?)
